sbnobj/Common/CRT/classes_def.xml

```diff
@@ -36,6 +37,10 @@
  <class name="art::Assns<sbn::crt::CRTHit, sbn::crt::CRTTrack,    void>"           />
  <class name="art::Wrapper< art::Assns<sbn::crt::CRTHit, sbn::crt::CRTTrack, void> >"           />
 
+ <class name="art::Assns<anab::T0,sbn::crt::CRTHit,void>" />
+ <class name="art::Assns<sbn::crt::CRTHit,anab::T0,void>" />
+ <class name="art::Wrapper<art::Assns<anab::T0,sbn::crt::CRTHit,void> >" />
+ <class name="art::Wrapper<art::Assns<sbn::crt::CRTHit,anab::T0,void> >" />
```

Comment:
Tradition demanded also the _art_ pointers to the objects (here, `art::Ptr<sbn::crt::CRTHit>`, since `anab::T0`'s one is defined in `lardataobj`) and their `std::pair` (involved in `art::Assns`) and their `std::vector`s to be added.
I believe I got away without that in the past, so I just recommend that a full test of write and read back is performed to validate this choice.

Reply:
It seems working and I am confirming it in this way. Maybe I will try adding more in next release.